src/main/resources/templates/requestInvite.html

```diff
@@ -11,7 +11,7 @@ <h2 class="heading-medium error-summary-heading" id="error-summary-heading-examp
                             There was a problem inviting this email address
                         </h2>
                         <ul class="error-summary-list">
-                            <li><a href="#email" th:text="${status}">status ...</a></li>
+                            <li><a href="#email" th:text="${status}" style="white-space: pre-line">status ...</a></li>
```

Comment:
Styling in HTML rather than in CSS - may cause inconsistency if someone expects to change this via a stylesheet but the inline style is applying.

Reply:
DONE.
Review comment is applied. Please refer PR:
https://github.com/Civil-Service-Human-Resources/identity-service/pull/196/files